templates/AureliaSpa/ClientApp/app/components/app/app.html

```diff
@@ -0,0 +1,14 @@
+<template>
```

Comment:
Inconsistent quote style in this file. In TypeScript let's use single-quotes. In HTML templates, I don't know if Aurelia community has a standard, but can you find out what it is and use that?